.github/workflows/ci.yml

```
@@ -81,6 +81,16 @@ jobs:
       - name: "Python type check"
         run: uvx mypy
 
+      - name: "Lint shell scripts"
+        uses: ludeeus/action-shellcheck@2.0.0
+        env:
+          # renovate: datasource=github-tags depName=koalaman/shellcheck
+          SHELLCHECK_VERSION: "v0.10.0"
+          SHELLCHECK_OPTS: --shell bash
+        with:
```

```suggestion
        with:
          version: ${{ env.SHELLCHECK_VERSION }}
```
otherwise the env var itself will not do anything.